pkg/integrations/v2/agent/agent.go

```diff
@@ -1,46 +0,0 @@
-// Package agent is an "example" integration that has very little functionality,
```

Comment:
Just curious: why was this taken out? Removing it means we have no v2 integrations now for testing, and my plan was to slowly migrate the existing v1 integrations to v2 over time.